crates/storage/src/schema/revision_0049.rs

```diff
@@ -0,0 +1,13 @@
+use anyhow::Context;
+
+pub(crate) fn migrate(tx: &rusqlite::Transaction<'_>) -> anyhow::Result<()> {
+    tx.execute_batch(
+        r"
+ALTER TABLE block_headers ADD COLUMN eth_l1_data_gas_price BLOB DEFAULT NULL;
+ALTER TABLE block_headers ADD COLUMN strk_l1_data_gas_price BLOB DEFAULT NULL;
+",
```

Comment:
Add `l1_da_mode`.

Reply:
Is there a reason we can't default to zero? i.e. are they actually missing for historic blocks, or just unused?

Reply:
We're defaulting to zero in `pathfinder_storage::connection::block::block_header()`.